Dockerfile

```diff
@@ -52,11 +52,13 @@ RUN mkdir -p ${NGINX_STATIC_DIR} && \
 RUN apt-get -qq update && \
     apt-get -qq -y install \
         cron \
+        gcc \
         gdal-bin \
         gettext \
         git \
         gosu \
         less \
+        libpq-dev \
```

Comment:
You don't need this nor gcc. Only `libpq5` is required for running psycopg2. All the on build packages are already installed in python (not slim) so you never need to type gcc.

We should make kpi do this same on build stuff at some point too for both consistency and smaller docker images.